src/utils/apiRequests.jsx

```diff
@@ -5,7 +5,12 @@ const ncNews = axios.create({
 });
 
 const getAllArticles = () => {
-  return ncNews.get("/articles").then(({ data }) => {
+  return ncNews.get(`/articles`).then(({ data }) => {
+    return data;
+  });
+};
+const getArticlesByTopic = (topic) => {
```

Comment:
So this is a perfectly acceptable way to do it BUT you could make things so much DRY-er by using axios params. It will basically let you reuse getAllArticles and in include the topic ONLY if it exists. Otherwise it will still just go to /articles. This would mean that you could just have the one AllArticles component (although might need renaming to match the new functionality) which is capable of returning either everything, or just articles by the topic passed. Give me a shout if you want to discuss, but the docs for axios params should be your first call.